lib/sdk/content/mod.js

```diff
@@ -56,6 +56,10 @@ function detach(modification, target) {
   else {
     let documents = iterator(modification);
     for (let document of documents) {
+      let window = document.defaultView;
+      // The window might have already gone away
+      if (!window)
+        continue;
```

Comment:
Just realized we have no tests for this file :( so I made [bug 1134839](https://bugzilla.mozilla.org/show_bug.cgi?id=1134839) about that.